backend/app/core/daily_journal_store.py

```
@@ -0,0 +1,322 @@
+        try:
+            await db.commit()
+        except IntegrityError:
+            # TOCTOU: get_by_date 확인 이후 LLM 호출이 긴 동안 동시 요청이 먼저 insert한 경우.
+            # UNIQUE(user_id, work_date) 위반 → 도메인 409로 매핑 (프론트는 기존 것 자동 로드).
+            await db.rollback()
+            raise DailyJournalError(
+                "already_exists",
+                "다른 요청이 먼저 해당 날짜의 통합 영농일지를 생성했습니다. "
+                "잠시 후 다시 확인해주세요.",
+                status=409,
+            )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)